### PR TITLE
Added message composer and dynamic loading of timeline events

### PR DESCRIPTION
**Pull request by arnav-t**
image
## Changes
- Added a basic message composer
  - Send messages either through the enter key or by clicking button 
  - Disable the composer and hide send button while a message is being sent or the composer is empty
  - Add newlines through shift+enter
- Make the messages load dynamically for the current room
- Minor CSS styling improvements

### CODE REVIEW

**Comment by arnav-t** [comment]
Added theme context and switched to using SCSS
image
image

File: components/client.jsx [hunk 1]
```diff
@@ -59,23 +75,31 @@ export default class Client extends Component{
...
     render() {
+        let theme = this.context;
+
         // Get current room ID
         let currentRoomId = this.state.room ? this.state.room.roomId : '';
         let homeserver = this.client.getHomeserverUrl();
 
         return (
-            <div className='client darker-bg'>
+            <div className={`client bg-priamry-${theme.theme}`}>
```
Comment (turt2live) [inline components/client.jsx]:
```suggestion
            <div className={`client bg-primary-${theme.theme}`}>
```

**Review by turt2live — APPROVED** [review]
Looks good overall!

**Review by benparsons — APPROVED** [review]
lgtm

**Comment by arnav-t** [comment]
Merged :smile:

**Pull request merged**